BeeAppy/src/screens/GeoMap.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  View,
+  Text
+} from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import MapView from 'react-native-maps';
+
+export default class GeoMap extends React.Component {
+  static navigationOptions = {
+    title: 'Map',
+  };
+  // Geolocation is built into React Native, takes initial state on line 18
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      latitude: 0,
+      longitude: 0,
+      error: null,
+    };
+  }
+
+  // Set geolocation to current position long and lat
+  componentDidMount() {
+    navigator.geolocation.getCurrentPosition(
+      (position) => {
+
+        this.setState({
+          latitude: position.coords.latitude,
+          longitude: position.coords.longitude,
+          error: null,
+        });
+      },
+    // print error if timed out or unable to find location
+      (error) => this.setState({ error: error.message }),
+      { enableHighAccuracy: true, timeout: 20000, maximumAge: 1000 },
+    );
+  }
+
+  giveOutput() {
+    console.log(JSON.stringify(this.state));
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+
+        <MapView
+          style={styles.map}
+          region={{ latitude: this.state.latitude, longitude: this.state.longitude,
+                           latitudeDelta: 0, longitudeDelta: 0}}
+        />
+
+          <Text>Latitude: {this.state.latitude}</Text>
+          <Text>Longitude: {this.state.longitude}</Text>
+        {this.state.error ? <Text>Error: {this.state.error}</Text> : null}
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
GeoMap is a class inherited from the previous geolocation base branch that jess worked on and I thought it was best to have this for reference